HyperwalletUISDK.xcodeproj/xcshareddata/xcschemes/Demo.xcscheme

```diff
@@ -38,6 +38,16 @@
                ReferencedContainer = "container:HyperwalletUISDK.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
No need to add these tests in Demo scheme

Reply:
Removed